docs/infrastructure/self-hosted/installation-chart/20-general-settings.md

```diff
@@ -114,8 +114,11 @@ mia-console:
 
 |                           Name                           |  Type   |                                                                      Description                                                                      | Default | Optional |
 | :------------------------------------------------------: | :-----: | :---------------------------------------------------------------------------------------------------------------------------------------------------: | :-----: | :------: |
-| `configurations.enableClustersAndEnvironmentsManagement` | boolean |                                                     Enables Clusters and Environments management                                                      | `true`  |    ✅     |
-|        `configurations.projectTemplateArchiveUrl`        | string  |                                                               New project template url                                                                |         |    ✅     |
+| `configurations.enableFastData` | boolean | Enables Fast Data configurator | `true`  |    ✅     |
+| `configurations.enableDebugArea` | boolean | Enables debug area in Console | `true`  |    ✅     |
+| `configurations.enableMergeConfiguration` | boolean | Enables Merge Configuration | `true`  |    ✅     |
```

Comment:
@davidebianchi these may have been wrongfully added with a merge

Reply:
There is in backend